contracts/governance/extensions/GovernorVotesQuorumFraction.sol

```diff
@@ -107,12 +110,12 @@ abstract contract GovernorVotesQuorumFraction is GovernorVotes {
         // Make sure we keep track of the original numerator in contracts upgraded from a version without checkpoints.
         if (oldQuorumNumerator != 0 && _quorumNumeratorHistory._checkpoints.length == 0) {
             _quorumNumeratorHistory._checkpoints.push(
-                Checkpoints.Checkpoint({_blockNumber: 0, _value: SafeCast.toUint224(oldQuorumNumerator)})
+                Checkpoints.Checkpoint224({_key: 0, _value: oldQuorumNumerator.toUint224()})
             );
         }
 
         // Set new quorum for future proposals
-        _quorumNumeratorHistory.push(newQuorumNumerator);
+        _quorumNumeratorHistory.push(uint256(clock()).toUint32(), newQuorumNumerator.toUint224());
```

Comment:
This seems to compile:

```suggestion
        _quorumNumeratorHistory.push(clock().toUint32(), newQuorumNumerator.toUint224());
```

Reply:
`Member "toUint32" not found or not visible after argument-dependent lookup in uint48.`

Reply:
That's just because we only have `using SafeCast for uint256` in this contract. We can change that to `using SafeCast for *` or simply add `uint48`.